api/stakeholder.go

```diff
@@ -188,16 +208,36 @@ func (r *Stakeholder) With(m *model.Stakeholder) {
 	r.Name = m.Name
 	r.Email = m.Email
 	r.JobFunction = r.refPtr(m.JobFunctionID, m.JobFunction)
+	r.Groups = []Ref{}
 	for _, g := range m.Groups {
 		ref := Ref{}
 		ref.With(g.ID, g.Name)
 		r.Groups = append(r.Groups, ref)
 	}
+	r.BusinessServices = []Ref{}
```

Comment:
hint: without this assignment, the nil list gets serialized to json as a null. Fixing this is intended to make the behavior consistent with the rest of the api.